stripe/src/main/java/com/stripe/android/model/CardMetadata.kt

```diff
@@ -12,8 +12,23 @@ internal data class CardMetadata internal constructor(
     internal data class AccountRange internal constructor(
         val binRange: BinRange,
         val panLength: Int,
-        val brandName: String,
-        val brand: CardBrand,
+        val brandName: BrandName,
         val country: String? = null
-    ) : StripeModel
+    ) : StripeModel {
+        val brand: CardBrand
+            get() = brandName.brand
+
+        internal enum class BrandName(
+            val code: String,
```

Comment:
`BrandName` and `code` here are kinda confusing to me. Maybe something like `BrandInfo`?

Reply:
Makes sense.